### DISCUSSION TITLE
[3.x] 404 for all actions in the form

### DISCUSSION THREAD

**Discussion by ArtMin96** [filamentphp/filament#7674] [Help] [answered]
When I search, it gives a 404

```php
class FormComponentName extends Component implements HasForms
{
    use InteractsWithForms;

    public ?array $data = [];

    public function mount(): void
    {
        $this->form->fill();
    }

    public function form(Form $form): Form
    {
        return $form
            ->schema([
                Grid::make([
                    'sm' => 1,
                    'md' => 3,
                ])
                    ->schema([
                        Select::make('from')
                            ->searchable()
                            ->getSearchResultsUsing(fn (string $search): array =>
                                Point::where('name', 'like', "%{$search}%")->limit(5)->pluck('name', 'id')->toArray()
                            )
                            ->getOptionLabelUsing(fn ($value): ?string => Point::find($value)->name),
                        Select::make('to')
                            ->searchable()
                            ->getSearchResultsUsing(fn (string $search): array =>
                                Point::where('name', 'like', "%{$search}%")->limit(5)->pluck('name', 'id')->toArray()
                            )
                            ->getOptionLabelUsing(fn ($value): ?string => Point::find($value)->name),
                    ]),
            ])
            ->statePath('data');
    }

    public function render(): View
    {
        return view('livewire.form-component-name');
    }
}
```

**Comment by pboivin** [comment]
What do you see in the browser's console?

**Reply by ArtMin96** [reply]
Sorry for the late reply.
![image](https://github.com/filamentphp/filament/assets/29732308/9c6a111a-d190-482c-8a0f-e4c62a375704)
I have no idea what could be causing the 404.

**Reply by ArtMin96** [reply]
https://github.com/filamentphp/filament/assets/29732308/7e6c8327-6e8a-4ba7-9a9d-ae4087ebf1b2

**Comment by pboivin** [comment]
Is your site really running on `http://localhost`? Could this be an issue with `APP_URL` in your env?

**Reply by ArtMin96** [reply]
No, everything is fine
```
APP_URL=http://localhost
```
![image](https://github.com/filamentphp/filament/assets/29732308/8c81b702-ae0c-4433-88b5-bc27b5e6e5c0)

**Reply by ArtMin96** [reply]
When I was debugging, the code does not enter ```getSearchResultsUsing(fn (string $search): array =>``` callback

**Reply by ArtMin96** [reply]
The problem is not in select. Form action also returns 404 after clicking.

**Comment by ArtMin96** [comment]
I didn't do anything but it started working

**Reply by pboivin** [reply]
Very strange... oh well, maybe you had a loose cable or something 😅️

**Reply by ArtMin96** [reply]
I did not notice, but the computer was turned off :joy:

**Comment by ArtMin96** [comment]
The same thing started to happen. Any action returns 404. form submit, ->live(), etc. After a while it started working again without intervention, then the same thing started again. It is not the first time that I use Filament. I can't understand what I can do to fix the problem.


**Searchable select:**

https://github.com/filamentphp/filament/assets/29732308/288e092e-2f91-4d43-b15c-553758da9759



**Live text input**


https://github.com/filamentphp/filament/assets/29732308/e62eb7d6-1a14-4a88-972c-69121cc2248b

```php
class SomeForm extends Component implements HasForms
{
    use InteractsWithForms;

    public ?array $data = [];

    public function mount(): void
    {
        $this->form->fill();
    }

    public function form(Form $form): Form
    {
        return $form
            ->schema([
                TextInput::make('phone')
                        ->afterStateUpdated(function (?string $state, ?string $old) {
                            dd($state, $old);
                        })
                        ->live()
            ])
            ->statePath('data');
    }
}
```

**Comment by ArtMin96** [comment, marked as answer]
The problem comes from this package: https://github.com/mcamara/laravel-localization/issues/880

Fix: [https://livewire.laravel.com/docs/upgrading#localization](https://livewire.laravel.com/docs/upgrading#localization)
```php
Route::group(['prefix' => LaravelLocalization::setLocale()], function ()
{
    // Your other localized routes...
 
    Livewire::setUpdateRoute(function ($handle) {
        return Route::post('/livewire/update', $handle);
    });
});
```
